build.sbt

```diff
@@ -33,12 +33,32 @@ developers += Developer(
 organizationName := "Lightbend Inc."
 startYear        := Some(2018)
 
-bintrayOrganization := Some("sbt")
-bintrayRepository   := "sbt-plugin-releases"
-
 enablePlugins(AutomateHeaderPlugin)
 scalafmtOnCompile := true
 
-// don't do any API docs
-doc / sources                := Seq()
-packageDoc / publishArtifact := false
```

Comment:
We can continue to skip publishing of sources and Scaladoc.

Reply:
Didn't this break publishing to scaladox (i.e. the issues behind https://github.com/lightbend/sbt-paradox-project-info/pull/32)?